ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ReplDumpTask.java

```diff
@@ -122,33 +128,38 @@ public String getName() {
   @Override
   public int execute() {
     try {
-      Hive hiveDb = getHive();
-      Path dumpRoot = new Path(conf.getVar(HiveConf.ConfVars.REPLDIR),
-              Base64.getEncoder().encodeToString(work.dbNameOrPattern.toLowerCase()
-                      .getBytes(StandardCharsets.UTF_8.name())));
-      Path previousDumpMetaPath = getPreviousDumpMetadataPath(dumpRoot);
-      Path previousHiveDumpPath =
-              previousDumpMetaPath != null ? new Path(previousDumpMetaPath, ReplUtils.REPL_HIVE_BASE_DIR) : null;
-      //If no previous dump is present or previous dump was loaded, proceed
```

Comment:
why not move the above statement in the function before  this

Reply:
Done